### ISSUE TITLE
GETS STUCK AT 1736 ITEMS SCANNED

### ISSUE THREAD

**Issue by 14paavang** [ThioJoe/YT-Spammer-Purge#374]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Well, I was trying to scan this video https://www.youtube.com/watch?v=4NqLdR48g3A , but I just got stuck at 1736 comments scanned.
<img width="960" alt="image" src="https://user-images.githubusercontent.com/81815334/149070282-3e772845-a2ee-45fa-9af4-b60631a6afce.png">

### Release version

latest non beta as of 12.1.2021 11:12AM IST

### Steps to reproduce

1. open the program
2. use scan specific video option
3. scan this video
4. https://www.youtube.com/watch?v=4NqLdR48g3A
5. us auto mode (1) 
6. it will get stuck at 1736

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
no error messdage is just hangs
```

### Screenshots

```shell
attached imn the main =body
```

**Comment by 14paavang** [comment]
i tried right clicking inside the window, nothing

**Comment by Firecul** [comment]
Huh, it does hang... strange.

![WindowsTerminal_Windows_PowerShell (10)](https://user-images.githubusercontent.com/34950371/149253392-b9f367cc-6e0b-49d0-81be-54868e671ffe.png)

That prob requires some debugging, I'd actually leave the issue open for @ThioJoe to take a peek at, just in case it comes up in another way.

I was using Windows .py v2.10.0 for reference.

**Comment by 14paavang** [comment]
i just saw the reply, i only closed it as I just had to wait for an hour while I ate dinner and when I came back it was fixed, still you might want to take a look at what is going on during that time, try to make it work quicker

**Comment by ThioJoe** [comment]
So it turns out it's this comment: https://www.youtube.com/watch?v=4NqLdR48g3A&lc=Ugz5YUx8XaBlvc_gY5V4AaABAg

It's not actually frozen completely but just really slowly going through each test. I'll have to figure out a way to mitigate it or optimize whichever tests are slowing it down the most.

**Comment by Firecul** [comment]
Make a new test, if there is more than 10 `.` in a row call it spam and move on :)

**Comment by ThioJoe** [comment]
Now fixed with 0d94ce6a6fb8241f0f32cfabe437dc463785a8c0

It would only happen when there were massive amounts of certain characters so was pretty rare I guess.
